tests/features/reports.feature

```diff
@@ -341,9 +341,19 @@ Scenario: Fail with system error after Step using wrong file name as trigger
 # I sign
 
 # ===============================
-# Reoprts for  Scenarios
+# Reoprts for Scenarios with warnings
 # ===============================
 
+@fast @rw1 @warning
+Scenario: Warn when there are too many names
+  Given the Scenario report should include:
+  """
+  The name "Uli Udo User Sampson Jr" has more than 4 parts, but 4 is maximum allowed. The test will set the name to "Uli Udo User Jr"
```

Comment:
I think there's a typo in here compared to `The name "${name_str}" has more than 4 parts, but 4 is the maximum allowed. The test will will set the name to "${name_parts.join(" ")}".`